src/JK/WebsiteBundle/Controller/IndexController.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace JK\WebsiteBundle\Controller;
+
+use JK\CommonBundle\Controller\AbstractController;
+use Symfony\Bundle\FrameworkBundle\Templating\EngineInterface;
+use Symfony\Component\HttpFoundation\Response;
+
+class IndexController extends AbstractController
+{
+    private $templating;
+
+    public function __construct( EngineInterface $templating)
+    {
+        $this->templating = $templating;
+    }
+    public function indexAction($name='Jauhien')
```

Comment:
Please configure all routes and controller via annotations
http://symfony.com/doc/current/book/routing.html#including-external-routing-resources